src/components/TodoCard.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="grid w-72 grid-cols-2 gap-6 rounded-lg border-2 border-black p-3 sm:w-[30rem]">
+    <h1 class="text-lg font-bold sm:text-xl">{{ todo.title }}</h1>
+    <BasePriority
```

Comment:
Please use a background mapper here, so you don't need to create 3 variant for the priority for example:
```
const backgroundMapper = {
'High' : 'bg-orange-600',
...
```